spec/data/duke/iee_records_with_941_field/multi_items_one_journal.xml

```diff
@@ -0,0 +1,222 @@
+    </datafield>
+    <datafield tag="655" ind1=" " ind2="7">
+      <subfield code="a">Periodicals.</subfield>
+      <subfield code="2">fast</subfield>
+      <subfield code="0">(OCoLC)fst01411641</subfield>
+    </datafield>
+    <datafield tag="776" ind1="0" ind2="8">
+      <subfield code="x">1899-5241</subfield>
+    </datafield>
+    <datafield tag="710" ind1="2" ind2=" ">
+      <subfield code="a">Uniwersytet Przyrodniczy w Poznaniu,</subfield>
+      <subfield code="e">issuing body.</subfield>
+    </datafield>
+    <datafield tag="850" ind1=" " ind2=" ">
+      <subfield code="a">DLC</subfield>
+    </datafield>
+    <datafield tag="906" ind1=" " ind2=" ">
+      <subfield code="a">JOURNAL</subfield>
+    </datafield>
+    <datafield tag="942" ind1=" " ind2=" ">
+      <subfield code="d">2012-11-18 06:42:58 US/Eastern</subfield>
+    </datafield>
+    <datafield tag="943" ind1=" " ind2=" ">
+      <subfield code="m">2024-08-19 11:24:06 US/Eastern</subfield>
+      <subfield code="k">System</subfield>
+      <subfield code="a">param</subfield>
+      <subfield code="j">2024-08-19 11:24:06 US/Eastern</subfield>
+      <subfield code="d">https://na05.alma.exlibrisgroup.com/view/uresolver/01DUKE_INST/openurl?u.ignore_date_coverage=true&amp;portfolio_pid=53911857610008501&amp;Force_direct=true</subfield>
+      <subfield code="g">62797757350008501</subfield>
+      <subfield code="i">System</subfield>
+      <subfield code="b">EZProxy</subfield>
+      <subfield code="n">2024-08-19 15:24:06</subfield>
+      <subfield code="u">e?u.ignore_date_coverage=true&amp;rft.mms_id=99119345153208501</subfield>
+      <subfield code="l"> Available from 2014 until 2015.</subfield>
+      <subfield code="c">false</subfield>
+      <subfield code="s">Available</subfield>
+      <subfield code="q">JOURNAL</subfield>
+      <subfield code="8">53911857610008501</subfield>
+    </datafield>
+    <datafield tag="943" ind1=" " ind2=" ">
+      <subfield code="m">2025-05-15 03:48:08 US/Eastern</subfield>
+      <subfield code="k">System</subfield>
+      <subfield code="a">param</subfield>
+      <subfield code="j">2024-06-25 23:28:59 US/Eastern</subfield>
+      <subfield code="d">https://na05.alma.exlibrisgroup.com/view/uresolver/01DUKE_INST/openurl?u.ignore_date_coverage=true&amp;portfolio_pid=53832770730008501&amp;Force_direct=true</subfield>
+      <subfield code="g">62797755010008501</subfield>
+      <subfield code="i">NON_SFX_CREATOR</subfield>
+      <subfield code="b">EZProxy</subfield>
+      <subfield code="n">2024-06-26 03:28:59</subfield>
+      <subfield code="u">e?u.ignore_date_coverage=true&amp;rft.mms_id=99119345153208501</subfield>
+      <subfield code="l"> Available from 2008.</subfield>
+      <subfield code="c">false</subfield>
+      <subfield code="s">Not Available</subfield>
+      <subfield code="8">53832770730008501</subfield>
+    </datafield>
+    <datafield tag="941" ind1=" " ind2=" ">
```

Comment:
For this and the other new `.xml` fixtures you're adding here, can we put the `941` datafield in the natural sequence where we'd expect it to appear in MARCXML (i.e., before the existing `942`)? It'd help with clarity/consistency - I was confused at first because I didn't see it.